prowler/providers/aws/services/iam/iam_service.py

```diff
@@ -717,7 +717,7 @@ def __get_access_keys_metadata__(self):
                             "AccessKeyMetadata"
                         ]
                 except ClientError as error:
-                    logger.error(
+                    logger.warning(
```

Comment:
and this one?

Reply:
Good catch, I've included both exceptions.